### PR TITLE
Updating Release Guidelines and Standards

### PR DESCRIPTION
**Pull request by damienjburks**

### CODE REVIEW

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/cmb.md:58]: I'm thinking this would be a good right after "purpose," overviewing the process before getting into details.

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/cmb.md:28]:
```suggestion
```

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/cmb.md:40]:
```suggestion
### Qualifications for Participation

Individuals of any background or experience level may participate in a review.

To approve or request changes, an individual must be an appointed CMB member in good standing.

CMB members are appointed by the [Delivery WG], through application or recruitment. If you have questions, please reach out to a current WG member or join a community call.

### Release Manager Qualifications

A release manager shall be a [Delivery WG] approver or a CMB member who has provided feedback on a previous release cycle.

Release managers are expected to demonstrate the following qualities:

- Strong written communication skills
- High attention to detail
- Commitment to process and protocol
- Ability to parse and relay complex feedback
- Fundamental knowledge of the domain featured in the release
- Availability and responsiveness during the release cycle (at least one month)

### Breach of Decorum

Members of the Change Management Board are expected to follow the [FINOS Community Code of Conduct](https://community.finos.org/docs/governance/code-of-conduct) at all times.

Appointments shall be permanently revoked in the following cases:

- Repeat disrespectful communication
- Repeat obstructive behavior such as vague or non-actionable feedback
- Repeat abandonment of a stated commitment
- Undermining the process, such as deliberately circumventing or disregarding documented norms
```

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/cmb.md:65]:
```suggestion
```

File: docs/governance/working-groups/delivery/charter.md
Comment (eddie-knight) [inline docs/governance/working-groups/delivery/charter.md:37]:
```suggestion
- Collaborate with other [WG]s to coordinate releases
```

File: docs/governance/community-guidelines/releases/README.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/README.md:11]:
```suggestion
The [Communications WG] has overall ownership of releases. They are responsible for setting the roadmap and communicating project progress to the wider community. All other [WG]'s should contribute to determining the content of a release.
```

Curious about the meaning behind this: "The Communications WG has overall ownership of releases."

File: docs/governance/community-guidelines/releases/README.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/README.md:29]:
```suggestion
There are two types of milestones:
```

File: docs/governance/community-guidelines/releases/README.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/README.md:32]:
```suggestion
2. Service Sets: When appropriate, the controls of multiple services can be scheduled for simultaneous release. 
```

"Bundling" would need to be in the form of a reference architecture, which we haven't finished designing yet.

File: docs/governance/community-guidelines/releases/README.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/README.md:44]:
```suggestion

### Release Flow Diagram
```

File: docs/governance/community-guidelines/releases/README.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/README.md:50]:
```suggestion
### Pull Request Validation
```

File: docs/governance/community-guidelines/releases/README.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/README.md:60]:
```suggestion
### Release Candidate Preparation
```

File: docs/governance/community-guidelines/releases/README.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/README.md:69]:
```suggestion
### Final Release
```

File: docs/governance/community-guidelines/releases/README.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/README.md:56]:
```suggestion
   - **[Duplication Reduction WG]** ensures that the content is unique and does not duplicate or mimic any existing standards or known catalogs.
```

This isn't really in scope for what DupRed has been doing, better to omit.

File: docs/governance/community-guidelines/releases/README.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/README.md:52]:
```suggestion
1. **Submission:** A contributor proposes a deliverable or change through a Pull Request (PR) to the CCC repository.
  - If a release manager has not been preemptively appointed for this deliverable, the [Delivery WG] should do so at this time.
```

File: docs/governance/community-guidelines/releases/README.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/README.md:53]: Need to refine this section...

We haven't been requiring that every PR be ready for release. Perhaps we should make an Issue Template and have that be the trigger for final review, instead of a PR?

**Review by eddie-knight — APPROVED** [review]
I have lots of notes, but wouldn't have been able to gather any thoughts at all without this starting point!

**Review by eddie-knight — CHANGES_REQUESTED** [review]
Correction— hit the approve button by mistake. DM me when you're reading to talk through these suggestions :fist:

File: docs/governance/community-guidelines/releases/README.md
Comment (sshiells-scottlogic) [reply docs/governance/community-guidelines/releases/README.md:11]: I think the initial thinking behind this was the comms WG would be the folk speaking to the users and stakeholders to get requests, maintain the roadmap and then be commuinciating back to the users/stakeholders so they would own the release in this regard.

File: docs/governance/community-guidelines/releases/cmb.md
Comment (damienjburks) [reply docs/governance/community-guidelines/releases/cmb.md:98]: i agree with this.

File: docs/governance/community-guidelines/releases/cmb.md
Comment (damienjburks) [reply docs/governance/community-guidelines/releases/cmb.md:63]: @eddie-knight we'll need to define an application process for those who are interested. maybe have an interview or something included.

File: docs/governance/community-guidelines/releases/cmb.md
Comment (sshiells-scottlogic) [inline docs/governance/community-guidelines/releases/cmb.md:78]: Should we add anything about a lack of engagement here? i.e. revoke their status if they don't engage at all. Or is this not needed?

File: docs/governance/community-guidelines/releases/cmb.md
Comment (rgriffiths-scottlogic) [inline docs/governance/community-guidelines/releases/cmb.md:13]: "comprises" or "is composed of" but not "is comprised of"! I favour "is composed of" in this situation.

File: docs/governance/community-guidelines/releases/cmb.md
Comment (rgriffiths-scottlogic) [inline docs/governance/community-guidelines/releases/cmb.md:9]: Who decides who's on the CMB? And how big is it? Does it need to be unanimous approval? Or a simple majority? Is there a constitution or other definition document for it?

File: docs/governance/community-guidelines/releases/cmb.md
Comment (rgriffiths-scottlogic) [inline docs/governance/community-guidelines/releases/cmb.md:32]: Ah, I see, this is where the operating policy is defined. Should it be here? Or in the same area as the charter for SteerCo, WG Leads and so on?

File: docs/governance/community-guidelines/releases/cmb.md
Comment (rgriffiths-scottlogic) [inline docs/governance/community-guidelines/releases/cmb.md:33]: Does stabilized mean "no activity"? What does it actually mean?

File: docs/governance/community-guidelines/releases/cmb.md
Comment (rgriffiths-scottlogic) [inline docs/governance/community-guidelines/releases/cmb.md:51]: 5 approvals out of a CMB of how many?

File: docs/governance/community-guidelines/releases/cmb.md
Comment (rgriffiths-scottlogic) [inline docs/governance/community-guidelines/releases/cmb.md:58]: I think it would make sense to capture the CMB definitions that lie within this document into its own separate charter doc and refer to that one.

**Review by eddie-knight — DISMISSED** [review]
LGTM— further improvements can come in follow-up PRs

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [reply docs/governance/community-guidelines/releases/cmb.md:58]: Good shout. Let's remember to add that in a follow-up

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [reply docs/governance/community-guidelines/releases/cmb.md:33]: Reasonably vague by intention here— the release manager should make a judgment call to determine whether opinions have been properly voiced and considered

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [reply docs/governance/community-guidelines/releases/cmb.md:9]: No max, the idea is to give the community an opportunity to review and approve— and give everyone confidence that at least a certain number of organizations have approved it as ready

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [reply docs/governance/community-guidelines/releases/cmb.md:13]:
```suggestion
The change management board is composed of a Release Manager and the body of reviewers.
```

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [reply docs/governance/community-guidelines/releases/cmb.md:32]: Good point... I think this is a good location for now though

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [reply docs/governance/community-guidelines/releases/cmb.md:78]: Shouldn't be necessary, as there is no community impact to low engagement from CMB members and they have no individual responsibilities

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/cmb.md:13]:
```suggestion
The change management board is composed of a Release Manager and the body of reviewers, both appointed by the [Delivery WG].
```

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [reply docs/governance/community-guidelines/releases/cmb.md:51]: no limit

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/cmb.md:102]:
```suggestion
```

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/cmb.md:10]:
```suggestion

## Process

The process followed by the CMB to manage changes includes:
	
1. **Proposal Submission**
   - Proposed changes are submitted for CMB review by contributors or working groups within the CCC project.
1. **Review Cycle**
   - The CMB reviews the changes based on the established guidelines and feedback from relevant working groups such as the [Security WG], [Delivery WG], and others.
1. **Approval or Request for Modifications**
   - After review, the CMB either approves the proposed changes for the next release candidate or requests modifications and additional feedback from the contributor or associated working group.
1. **Final Approval and Release**
   - Upon receiving approval, the release manager compiles the final release package, and the CMB confirms the official release of the updated framework.
   
```

File: docs/governance/community-guidelines/releases/cmb.md
Comment (eddie-knight) [reply docs/governance/community-guidelines/releases/cmb.md:51]:
```suggestion
- A release cannot proceed without a minimum of 5 approvals; members are encouraged to help meet this threshold by approving, requesting changes, or contributing to discussion around open change requests.
```

File: docs/governance/community-guidelines/releases/versioning.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/versioning.md:28]:
```suggestion
Releases should happen, at most, one time per month. This schedule ensures a manageable release cadence and maintains the stability of our artifacts. For more information about the releases, please refer to this document: [Releases](/docs/governance/community-guidelines/releases.md)
```

Releases should happen, at most, one time per month. This schedule ensures a manageable release cadence and maintains the stability of our artifacts. For more information about the releases, please refer to this document: [Releases](/docs/governance/community-guidelines/releases.md)

File: docs/governance/community-guidelines/releases/versioning.md
Comment (eddie-knight) [inline docs/governance/community-guidelines/releases/versioning.md:28]:
```suggestion
Releases should happen, at most, one time per month. This schedule ensures a manageable release cadence and maintains the stability of our artifacts. For more information about the releases, please refer to this document: [Releases](./README.md)
```

**Review by eddie-knight — APPROVED** [review]

**Pull request merged**
